src/GovUk.Education.ExploreEducationStatistics.Content.Services/ViewModels/PublicationCacheViewModel.cs

```diff
@@ -0,0 +1,29 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Services.ViewModels
+{
```

Comment:
Could change this to use file namespacing

Reply:
Thanks, I've made this change.